### ISSUE TITLE
begging for the  most simple example of using channels or queues with interpreters_pep_734 ?

### ISSUE THREAD

**Issue by stonebig** [ericsnowcurrently/interpreters#13]
it's hard to decipher through old iterations of implementation of _xxinterpreters to get something working.

so the begging for tiny and simple examples

**Comment by temeddix** [comment]
Agreed, putting some example code in README would be nice so that more people can get involved and provide feedback.

Hope PEP734 makes its way into the stdlib soon!
